### DISCUSSION TITLE
Datagrid scroll to top by click or event

### DISCUSSION THREAD

**Discussion by SvenBangert** [vmware/clarity#5902] [Q&A] [answered]
Hi there,

is there any method to scroll the datagrid up to top by click or event? There ist a report of a bug here in this forum like "grid scrolls up at single action" - so it seems to be possible to scroll to top - if accidently or not :-).

Any idea?

Thanks a lot!

**Comment by Jinnie** [comment]
We need a reproduction of this. Can't comment without seeing the problem live.
These are the stackblitz templates we use: https://stackblitz.com/@clr-team

**Comment by SvenBangert** [comment, marked as answer]
Hi and thanks for response.
I solved it by myself with following code.

```
public scrollToTop() {
  document.getElementsByClassName('datagrid')[0].scrollTop = 0;
}
```
